src/Couchbase.Lite.Shared/Support/Android/AndroidConsoleLogger.cs

```diff
@@ -32,7 +32,7 @@ internal sealed class AndroidConsoleLogger : IConsoleLogger
 
         public LogDomain Domains { get; set; } = LogDomain.All;
 
-        public LogLevel Level { get; set; } = LogLevel.Warning;
+        public LogLevel Level { get; set; } = LogLevel.None;
```

Comment:
Because otherwise the logging is going to be in two places right?  I disabled the old api by default so that the new one could be used.  I don’t see any benefit in keeping the old one defaulted to warning when the new one has a default of warning.